tests/test_logging/test_logging.py

```diff
@@ -0,0 +1,37 @@
+import subprocess
+import os
+import sys
+from shutil import rmtree
+import pytest
+import re
+
+def capture(command,instream=None):
+    proc = subprocess.Popen(command,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            stdin=instream
+                            )
+    out, err = proc.communicate()
+    return out, err, proc.returncode
+
+
+def test_logging_to_file(python_version):
+    """Test logging Terminal output to a log file."""
+    path_basic_scene = os.path.join("tests", "tests_data", "basic_scenes.py")
+    expected=['INFO', 'Read', 'configuration', 'files:', 'config.py:92', 'INFO',
```

Comment:
**Chore :** Please document the workaround you did (the word thing because rich output is different depending on the OS)

Reply:
Done :)